package.json

```diff
@@ -12,10 +12,9 @@
     "packages/*"
   ],
   "scripts": {
-    "build": "yarn run build:source && yarn run build:types",
+    "build": "yarn ts-bridge --project tsconfig.build.json --verbose",
     "build:clean": "rimraf dist '**/*.tsbuildinfo' && yarn build",
     "build:docs": "yarn workspaces foreach --all --no-private --parallel --interlaced --verbose run build:docs",
-    "build:source": "yarn workspaces foreach --all --no-private --parallel --interlaced --verbose run build",
     "build:types": "tsc --build tsconfig.build.json --verbose",
     "build:watch": "yarn run build --watch",
```

Comment:
Does `ts-bridge` have a watch mode?

```suggestion
```

Reply:
Good catch, it doesn't. Removed in 699b56a5bfb6704f38f41f4b7c52d336d9bb0b0e.